composer.json

```diff
@@ -18,41 +18,38 @@
       "role": "Developer"
     }
   ],
-  "minimum-stability": "stable",
-  "prefer-stable": true,
-  "support": {
-    "issues": "https://github.com/widoz/event-listeners-collection/issues"
-  },
-  "autoload": {
-    "classmap": [
-      "src/"
```

Comment:
Why removing this? I thought it could be a good thing when you want to build an optmized autoloader map, unless `composer dump -o` does that without the needs of a `classmap`?

Reply:
In PSR-4 it is up to the user:

- files based
- `-o` hardcoded in the autoloader + file based
- `-a` only hardcoded classes

Reply:
I read the documentation again and yes I came to the conclusion that a lib shouldn't take care of optimization but application should